erpnext/public/js/controllers/transaction.js

```diff
@@ -1911,9 +1910,14 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 	set_query_for_item_tax_template: function(doc, cdt, cdn) {
 		var item = frappe.get_doc(cdt, cdn);
 		if(!item.item_code) {
-			frappe.throw(__("Please enter Item Code to get item taxes"));
+			if (doc.company) {
+				return {
+					filters: {company: doc.company}
+				};
+			} else {
+				return {};
+			}
```

Comment:
```suggestion
          let company_filter = {filters: {company: doc.company}};
          return doc.company ? company_filter : {};
```

Reply:
Now you're just being pedantic. Has your change been tested? Because mine has and I'm not testing yours.
You should be grateful that I have found, fixed and tested a serious regression and bug in your product.